wpilibc/src/main/native/include/frc/Compressor.h

```diff
@@ -113,34 +121,65 @@ class Compressor : public wpi::Sendable,
   void Disable();
 
   /**
-   * Enable compressor closed loop control using digital input.
+   * Enables the compressor in digital mode. The compressor will turn on when
+   * the pressure switch indicates that the system is not full, and will turn
+   * off when the pressure switch indicates that the system is full.
    */
   void EnableDigital();
 
   /**
-   * Enable compressor closed loop control using analog input. Note this is only
-   * for use with the REV Analog Pressure Sensor.
+   * If supported by the device, enables the compressor in analog mode. This
+   * mode uses an analog pressure sensor to cycle the compressor. The compressor
```

Comment:
Mention that the analog pressure sensor used is one connected to port 0 of the PH.